src/DssSpell.t.sol

```diff
@@ -354,33 +373,77 @@ contract DssSpellTest is DssSpellTestBase {
         // assertEq(dai.balanceOf(wallets.addr("DAIF_WALLET")), prevBalance + 67_863 * WAD);
     }
 
-    function testFlash() public {
+    function _setupL2s() internal {
+        vm.makePersistent(address(spell), address(spell.action()));
+
+        string memory root = string.concat(vm.projectRoot(), "/lib/dss-test");
+        config = ScriptTools.readInput(root, "integration");
+
+        rootDomain = new RootDomain(config, getRelativeChain("mainnet"));
+        optimismDomain = new OptimismDomain(config, getRelativeChain("optimism"), rootDomain);
+        arbitrumDomain = new ArbitrumDomain(config, getRelativeChain("arbitrum_one"), rootDomain);
```

Comment:
after thinking about it some I think these would be better in their specific tests
```suggestion
```